### ISSUE TITLE
Localization of text messages coming from api-server

### ISSUE THREAD

**Issue by taimir** [kubernetes/dashboard#974]
#### Issue details

Currently text messages returned by the kubernetes core (in particular `api-server`) are not localized. This is because the api-server is an external system component, and the dashboard localization framework does not apply to it.

A decision needs to be made about how to display these messages, and whether to treat them in any particular way.
##### Environment

```
Dashboard version: v1.1.0
Kubernetes version: v1.2.5
```
##### Steps to reproduce

Clone dashboard. Start with `gulp serve:prod`. Use `ja` as an example locale (different than `en`) for your browser.
##### Observed result

In general, error messages and status messages that come directly from the k8s API-server are still displayed in plain English. This also applies to short status strings, e.g. `Running` for a Pod.
##### Expected result

The expected behavior is a matter of further discussion.

**Comment by taimir** [comment]
Off the top of my head:
* Complex error messages will be extremely hard to localize on the side of `dashboard`. To my knowledge, currently the k8s api-server does not support any type of localization on its own. Correct me if I'm wrong.

* It might be a good idea to not display long status / error / warning messages from the api-server directly, but to hide them behind some sort of generic "proxy" text messages. This would have the added benefit of better control of the text flow on dashboard.

* For short status text, such as the status of a pod that's returned by the api-server, localization on dashboard's side might be feasible. However, some sort of mappings will need to be defined. Currently it is not clear to me how many such status "enum values" exist in k8s (regarding all resources), so I cannot assess the actual overhead of this additional change.

**Comment by taimir** [comment]
@cheld @bryk @floreks @maciaszczykm @digitalfishpond

**Comment by bryk** [comment]
> It might be a good idea to not display long status / error / warning messages from the api-server directly, but to hide them behind some sort of generic "proxy" text messages. This would have the added benefit of better control of the text flow on dashboard.

Statuses, types, etc. can be easily translated by the "proxy". They are mostly enums, so we can just hardcode translations in the UI. I've seen projects that do this.

**Comment by bryk** [comment]
Regarding free form text messages, like descritpion of an error event. We can either ignore that or provide our own messages for errors and events, rather than taking them directly from the api. It is more work, but certainly doable.

**Comment by GavinSong123** [comment]
How to change 'ja' on dashboard? Is there a config file?

**Comment by cheld** [comment]
It is defined by the browser locale. Change the browser to Japanese

**Comment by GavinSong123** [comment]
@cheld ok, thx.

**Comment by zouyee** [comment]
If you resolved the problem,please closed it

**Comment by maciaszczykm** [comment]
@zouyee AFAIK it is not fixed yet.
